packages/models/src/lib/table.ts

```diff
@@ -1,20 +1,21 @@
 import { z } from 'zod';
 import { primitiveValueSchema } from './implementation/schemas';
 
+export const tableAlignmentSchema = z.enum(['l', 'c', 'r'], {
+  description: 'Cell alignment (l:left, r:right, c:center)',
+});
+export type TableAlignment = z.infer<typeof tableAlignmentSchema>;
```

Comment:
Why not just make the enum values self-descriptive?

```suggestion
export const tableAlignmentSchema = z.enum(['left', 'center', 'right'], {
  description: 'Cell alignment',
});
export type TableAlignment = z.infer<typeof tableAlignmentSchema>;
```

Reply:
renamed